### DISCUSSION TITLE
SEC API

### DISCUSSION THREAD

**Discussion by Spieledmilk26** [community/community#37056] [Other Feature Feedback, Questions, & Ideas]
Anyone familiar with using the SEC EDGAR API and have resources they can share? I want to pull 10-K reports and extract data. Most of the videos I have seen are hard to follow. Any help would be greatly appreciated

**Comment by ossanna16** [comment]
Hi @Spieledmilk26,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on SEC API. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
